doc/source/ray-contribute/docs.ipynb

```diff
@@ -38,7 +38,6 @@
     "Follow the instructions in [Building Ray from Source](https://docs.ray.io/en/master/ray-contribute/development.html#building-ray-from-source) up to and including the following:\n",
     "1. [Fork the Ray repository](https://docs.ray.io/en/master/ray-contribute/development.html#fork-the-ray-repository)\n",
     "2. [Clone the forked repository](https://docs.ray.io/en/master/ray-contribute/development.html#fork-the-ray-repository) to your local machine\n",
```

Comment:
This section seems a bit overkill. It says to follow the instructions for building ray from source, but is really just pointing to the single step of forking ray.

Also the headers seem incorrect right now. "Building the Ray documentation" only includes the logic for forking Ray, and "Install Dependencies" also includes the step for building the docs.

Maybe a nested structure would be better?
- Building the Ray documentation
   - Fork Ray
   - Install Dependencies
   - Build documentation

Reply:
Great feedback and suggestions. Thank you.